metaflow/plugins/conda/conda_flow_decorator.py

```diff
@@ -37,7 +37,7 @@ class MyFlow(FlowSpec):
                 'python': None,
                 'disabled': None}
 
-    def flow_init(self, flow, graph, environment, datastore, logger):
+    def flow_init(self, flow, graph,  environment, datastore, logger, echo, options):
```

Comment:
Need to make the same change **[here](https://github.com/Netflix/metaflow/blob/master/metaflow/plugins/aws/step_functions/schedule_decorator.py)**

Reply:
Good catch. Missed it in all the rebasing and intro of step functions. I'll check if I forgot another one too.